README.md

```diff
@@ -338,8 +337,58 @@ mallet> getReceipt()
   gasUsed: 28263,
   contractAddress: null,
   logs: [],
-  status: '0x00',
-  returnData: '0xc0' }
+  statusCode: '0x00',
+  status: true,
+  returnData: [],
+  rawReturnData: '0xc0' }
+```
+
+#### `iele.constantCall`
```

Comment:
why not simply `iele.call`?

Reply:
Well... I would be fine with that, as would be people familiar with JSON-RPC/web3. But my understanding is we're also targeting users who aren't